### DISCUSSION TITLE
Problem with only N64 and Genesis

### DISCUSSION THREAD

**Discussion by JoeBuk724** [sanni/cartreader#954] [Need help]
Hi! I recently built the Hardware version 5 with the V13.1 code. I’m having a problem with getting it to read Genesis and N64 games. All the other slots read/dump just fine, I loaded the ROMs up elsewhere to make sure they worked. 

The Genesis did work when I first built it (about 2 weeks ago) I dumped my Sonic 1 cartridge and it works just fine. I have no idea what I did to it (I did take it apart a couple times though) since then that made it start having problems in only those two spots.

So when I try to load a Genesis game it usually gives me a goofy title, sometimes it’s numbers and letters, sometimes it’s blank and sometimes it’s close but misspelled like “SONICTHEHEGDGE” I’ve tried a handful of games and they all do the same. I did notice if the switch is set to 3V it tends to have an easier time almost getting the title right instead of it being blank, I’m sure it’s nothing but just thought I’d mention it.

As for the N64 I only have one game to test with atm (WWF War Zone) but it always comes up with “GAMEPAK ERROR”. I’m going to try and get another game to make sure it’s not just the cart.

I thought it may have been the slots themselves so I ordered replacements and swapped them out, but the problem persists. I’ve cleaned the cartridges several times but that doesn’t seem to help either. All my solder points look good too.

I’ve re-tested the other slots to make sure they all still work and they do. I have no idea what my issue is. Any help would be greatly appreciated.

**Comment by JoeBuk724** [comment]
Update: So I made a second top board and everything seems to be working fine now. So at some point i must have messed the original up somehow. So if anyone runs into a similar problem it may be worth a try. Peace!

**Comment by cmcm711** [comment]
I've had to do this a couple times before.
